tests/foreman/virtwho/test_ui_virtwho.py

```diff
@@ -15,19 +15,27 @@
 :Upstream: No
 """
 from fauxfactory import gen_string
+
+from airgun.session import Session
 from robottelo.config import settings
 from robottelo.decorators import (
     fixture,
     tier2
 )
 
 from .utils import (
+    add_configure_option,
+    delete_configure_option,
     deploy_configure_by_command,
     deploy_configure_by_script,
     get_configure_id,
     get_configure_file,
     get_configure_option,
     get_configure_command,
+    get_virtwho_status,
+    restart_virt_who_service,
```

Comment:
could you rename it to `restart_virtwho_service`?

Reply:
ok, modify this with next commit